scripts/us_bls/cpi/generate_csv.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''
+Downloading and converting BLS CPI raw csv files to csv files of two columns:
+"date" and "cpi", where "date" is of the form "YYYY-MM" and "cpi" is numeric.
+
+Usage: python3 generate_csv.py
+'''
+
+
+from io import StringIO
+
+import requests
+import pandas as pd
+
+
+# Dict from series names to download links
+CSV_URLS = {
```

Comment:
Should we make these constants private by prepending an underscore?

Reply:
I don't have the intention of disallowing other scripts to access them, but I am going to wrap them in frozendicts so they can't be modified

Reply:
wrapped in frozendicts